src/DatabaseL2.php

```diff
@@ -461,24 +461,28 @@ public function applyEvents(L1 $l1)
             return null;
         }
 
-        $applied = 0;
         try {
             $sql = 'SELECT "event_id", "pool", "address", "value", "created", "expiration"'
                 . ' FROM ' . $this->eventsTable
                 . ' WHERE "event_id" > :last_applied_event_id'
-                . ' AND "pool" <> :exclude_pool'
```

Comment:
This was changed to make both L2 drivers to behave consistently.
Though this change is causing small performance regressions, it is behaving as the StaticL2 does at the moment.
There is a non-merged implementation that makes them like the DB [here](https://github.com/ndobromirov/lcache/commit/c09246ad604040bd2bcad65274f0bcb93d07f6a0)

I will maintain both branches until it is decided how L2::applyEvents should behave and merge the correct version.